src/components/Reports/Reports.js

```diff
@@ -48,6 +51,10 @@ class Reports extends React.Component {
     this.setState({ pageSelected: pageFound });
   };
 
+  changePeriod = (id) => {
+    this.setState({ periodSelectedDays: id });
```

Comment:
I think that `id` is not the best name for a number of days.

Reply:
yes i forgot to change. Thanks!